### PR TITLE
Update placeholderplaceholderplaceholder occurrences to our doc site

### PR DESCRIPTION
**Pull request by eranelbaz**
Part of #165
Updated docs and example

## Target Release

1.6.0

### NEW FEATURES | UPGRADE NOTES | ENHANCEMENTS | BUG FIXES | EXPERIMENTS

 

-

### CODE REVIEW

File: internal/command/e2etest/automation_test.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ import (
 
 // The tests in this file run through different scenarios recommended in our
 // "Running Terraform in Automation" guide:
-//     https://www.placeholderplaceholderplaceholder.io/guides/running-terraform-in-automation.html
+//     https://developer.hashicorp.com/terraform/tutorials/automation/automate-terraform
```
Comment (eranelbaz) [inline internal/command/e2etest/automation_test.go:19]: we currently do not have such a tutorial, we can keep the placeholder for now if we would like to

File: internal/command/meta_backend_migrate.go [hunk 2]
```diff
@@ -982,7 +982,7 @@ the error above and try again.
 const errTFCMigrateNotYetImplemented = `
 Migrating state from Terraform Cloud to another backend is not yet implemented.
 
-Please use the API to do this: https://www.placeholderplaceholderplaceholder.io/docs/cloud/api/state-versions.html
+Please use the API to do this: https://www.terraform.io/docs/cloud/api/state-versions.html
```
Comment (eranelbaz) [inline internal/command/meta_backend_migrate.go:985]: we don't have cloud docs

**Review by RLRabinowitz — COMMENTED** [review]
I see that there are still some usages of `placeholderplaceholderplaceholder` in the code in this branch:

1. [errTFCMigrateNotYetImplemented](https://github.com/opentofu/opentofu/blob/efe8ff6b0aec31f3862760c64bcc4405f4dffe62/internal/command/meta_backend_migrate.go#L985C20-L985C20) - This revolves around migrating from a `cloud` to any other `backend`, and is not bound to terraform. You can adjust the wording accordingly and remove the link
2. [tfcInputBackendMigrateMultiToMulti](https://github.com/opentofu/opentofu/blob/efe8ff6b0aec31f3862760c64bcc4405f4dffe62/internal/command/meta_backend_migrate.go#L1002-L1015) - Same as above, you can change the wording to not mention this revolves around a `cloud` in general, and remove the link
3. In [apt.mdx](https://github.com/opentofu/opentofu/blob/efe8ff6b0aec31f3862760c64bcc4405f4dffe62/website/docs/cli/install/apt.mdx#L21) and [yum.mdx](https://github.com/opentofu/opentofu/blob/efe8ff6b0aec31f3862760c64bcc4405f4dffe62/website/docs/cli/install/yum.mdx#L33-L44) - The guides are currently wrong, and it seems like we can't install `tofu` using `yum`/`apt`/`dnf`. We should probably create an issue for that, the issue should include changing the docs as well
4. [Comment in automation_tests.go](https://github.com/opentofu/opentofu/blob/efe8ff6b0aec31f3862760c64bcc4405f4dffe62/internal/command/e2etest/automation_test.go#L17-L19) - You can just remove this comment

**Comment by eranelbaz** [comment]
> In [apt.mdx](https://github.com/opentofu/opentofu/blob/efe8ff6b0aec31f3862760c64bcc4405f4dffe62/website/docs/cli/install/apt.mdx#L21) and [yum.mdx](https://github.com/opentofu/opentofu/blob/efe8ff6b0aec31f3862760c64bcc4405f4dffe62/website/docs/cli/install/yum.mdx#L33-L44) - The guides are currently wrong, and it seems like we can't install tofu using yum/apt/dnf. We should probably create an issue for that, the issue should include changing the docs as well

Wrote that on the issue

File: internal/command/meta_backend_migrate.go [hunk 3]
```diff
@@ -1005,7 +1003,7 @@ configuration (e.g. production, staging, development), Terraform Cloud workspace
 across all configurations used within an organization. A typical strategy to start with is
 <COMPONENT>-<ENVIRONMENT>-<REGION> (e.g. networking-prod-us-east, networking-staging-us-east).
 
-For more information on workspace naming, see https://www.placeholderplaceholderplaceholder.io/docs/cloud/workspaces/naming.html
```
Comment (RLRabinowitz) [inline internal/command/meta_backend_migrate.go:1008]: I would just remove this part, as the link to docs you've listed here doesn't have to do with what is mentioned here (naming recommendations for `cloud` workspaces)

**Review by RLRabinowitz — APPROVED** [review]

**Pull request merged**